### PR TITLE
LOCUS-42 - add jsonschema2pojo.jsonschema2pojo-core

### PR DESCRIPTION
**Pull request by pleacu**

### CODE REVIEW

**Comment by pleacu** [comment]
@mickaelistria  - resubmitted for inclusion into 1.3.0-SNAPSHOT - still needs to be tested by Data Trans.

File: plugins/org.jboss.tools.locus.jsonschema2pojo/0.4.13/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.jboss.tools.locus.jsonschema2pojo.jsonschema2pojo-core
+Bundle-Version: 0.4.13
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6
```
Comment (mickaelistria) [inline plugins/org.jboss.tools.locus.jsonschema2pojo/0.4.13/META-INF/MANIFEST.MF]: You'll need some Export-Package directive to make it usable to clients. Also, doesn't this bundle have dependencies?

Comment (pleacu) [inline plugins/org.jboss.tools.locus.jsonschema2pojo/0.4.13/META-INF/MANIFEST.MF]: Hold off on the PR for a bit - I'll rebuild Data Transformation with an updated JBTIS TP referencing my local copy of locus (updated for exports).  thkx

**Comment by nickboldt** [comment]
Please squash this when you're happy w/ it.

**Comment by pleacu** [comment]
merged Nick's TP changes into my dev branch - will re-issue a PR after https://github.com/jbosstools/jbosstools-locus/pull/45 is pushed

**Pull request closed without merging**